mteb/tasks/Classification/ben/SentimentAnalysisBengali.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class SentimentAnalysisBengali(AbsTaskClassification):
```

Comment:
Please update all references as well, including the model runs.
```suggestion
class BengaliSentimentAnalysis(AbsTaskClassification):
```